be/db/dev-db/docker-compose.yaml

```diff
@@ -0,0 +1,16 @@
+version: '3.1'
+services:
+  db:
+    image: postgres:latest
+    restart: always
```

Comment:
Why? The db container might fall into a crash loop with this policy

Reply:
but it also will restart when will go down. You want either container that will crash loop when somethings wrong. Or container that will die forever when some unexpected happens. Usual solution is restart always. Turn it on and off fixes lot of issues.